test/JDBC/expected/sys-type_info_internal-vu-verify.out

```diff
@@ -0,0 +1,43 @@
+
+
+-- Simple test to make sure to catch any changes to the sys.type_info_internal view.
+-- If the changes are intentional the expected result of this test should be modified.
+SELECT pg_type_name, tsql_type_name
```

Comment:
This does not guarantee `type_info_internal` matches `type_infos`. Please add a sanity check function into `babelfish_integrity_checker` (Ref: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/713)

Reply:
I removed the sys.type_info_internal view and changed the implementation of the sys.types so that it uses the sys.babelfish_typecode_list function instead of the sys.type_info_internal view. The performance is the same.

Therefore, we don't have to worry about keeping it in sync with the type_infos any more.